New package: Kerchunk v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Kerchunk
- Repository: https://github.com/JuliaIO/Kerchunk.jl
- Created by: @asinghvi17
- Version: v0.1.0
- Commit: 2d73c2b5e0464409d7883cbe1002b9391a1f4eb4
- Reviewed by: @asinghvi17
- Reference: https://github.com/JuliaIO/Kerchunk.jl/commit/2d73c2b5e0464409d7883cbe1002b9391a1f4eb4#commitcomment-146744391
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd63b437cc8e55c2bbe2e4047c8fcd8b486002b5c0e46accf257eba5fac0f7c56be2f905a469af59692692a7271b79435c9dcb8f895954e77eef9eeafdaadea80e116059c243964a77ca9206304949fee3f184f5edb44e379c7c0d7bab093b3900135177b792bdcbc04261668b50fc26f526cd6e9609e201698e3c58f27016b6fa54a876fb0d38e7f41e6fb587e436b8223d51297bfc012bf3ed243a4f44a517ab5c8992c624e4c316b6b2112e3ca8c2e1d -->

FriesischScott commented:
In the README the author states that the package is not actually compatible with *Zarr.jl* version 0.9  but relies on an unreleased feature branch.

asinghvi17 commented:
[Noblock] to be more specific: the package itself will work for many catalogs, but some that are generated from netcdf files require Zarr codecs that are on that branch.  I could copy those to the package for now and remove the requirement, though.  Would that make sense?

JuliaRegistrator committed:
New package: Kerchunk v0.1.0

UUID: 12c09fd5-fe6a-4e79-8f42-b31f49215243
Repo: https://github.com/JuliaIO/Kerchunk.jl.git
Tree: e0341d869ef8e7e1e75fd5717a8b4ecd90113930

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

asinghvi17 commented:
[noblock] I've just updated the repo to remove the dependence on an unreleased branch of Zarr.jl. @FriesischScott does this work for you?
